cea/interfaces/dashboard/base/static/build/js/glossary_search.js

```diff
@@ -0,0 +1,27 @@
+const docsUrl = 'https://city-energy-analyst.readthedocs.io/en/latest/';
+
+$('#glossary').autocomplete({
+    serviceUrl: '/glossary_search',
+    dataType: 'json',
+    groupBy: 'category',
+    width: 'flex',
+    forceFixPosition: true,
+
+    transformResult: function(response) {
+        return {
+            suggestions: $.map(response, function(dataItem) {
+                return { value: dataItem.VARIABLE, data: { category: dataItem.SCRIPT,
+                        description: dataItem.DESCRIPTION, locator: dataItem.LOCATOR_METHOD} };
+            })
+        };
+    },
+
+    formatResult: function (suggestion, currentValue) {
+      return '<div>'+suggestion.value+'</div><small>'+suggestion.data.description+'</small>';
+    },
+
+    onSelect: function (suggestion) {
+        var type = 'input';
```

Comment:
@daren-thomas is it possible to add a column in `naming.csv` to indicate if the variable is from an `input` or `output` script? Because right now I have a glossary search bar where users can search for the variable and will be brought to the glossary in our docs for more information. 

Currently I am only able to generate part of the URL based on the information given in `naming.csv` since our docs separate the methods based on `input` or `output`  methods
e.g. https://city-energy-analyst.readthedocs.io/en/latest/input_methods.html#get-zone-geometry
https://city-energy-analyst.readthedocs.io/en/latest/output_methods.html#get-building-supply

Reply:
I'll take a look!

Reply:
sure. working on it. the doc string by @Jack-Hawthorne gave a hint (when script is set to the basename of the file) but i'm updating it to just be '-' instead, easy-peasy). i'll post my changes as soon as i've tested.

Reply:
done. it works now.